src/bootstrap/README.md

```diff
@@ -331,6 +331,21 @@ are:
   `Config` struct.
 * Adding a sanity check? Take a look at `bootstrap/sanity.rs`.
 
+If you make a major change, please remember to:
+
++ Update `VERSION` in `src/bootstrap/main.rs`.
+* Update `version = N` in `config.toml.example`.
+* Add an entry in `src/bootstrap/CHANGELOG.md`.
+
+A 'major change' includes
+
+* A new option or
+* A change in the default options.
+
+Changes that do not affect contributors to the compiler or users
+building rustc from source don't need an update to `VERSION` (although they
```

Comment:
Hm, what would an example of such a change be? It feels like if it affects no one, it should not be in the changelog...

Reply:
Good point, I'll remove that bit.

Do you still think this should warn when a new configuration option is added? That's not a breaking change, although we still might want to tell people about it.